opendatahub-io/odh-dashboard
# Update ovms with gpu support and remove the other template

## Description


Closes #1989

## How Has This Been Tested?



1. Delete the old templates
2. Deploy the new templates
3. Create a new ovms model with the new Custom Serving Runtime
4. **NOT TESTED IN A GPU CLUSTER, WE SHOULD DO THAT AT ONE POINT BEFORE THE RELEASE**

## Test Impact


Not needed, manifest files, we'll might include something for e2e testing in the future.

## Request review criteria:



Self checklist (all need to be checked):

If you have UI changes: 


After the PR is posted & before it merges:
